.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
+<<<<<<< HEAD
+    "python.linting.enabled": false
+=======
+    "todo-tree.flat": false,
+    "todo-tree.expanded": true,
+    "npm-scripts.showStartNotification": false
+>>>>>>> 82b1904cd94eab193a95b089863079fbebbe174e
+}
```

Comment:
This file appears to have been committed with an unresolved conflict

Reply:
I'll add it to the gitignore tmrw, thx